spec/formatting.md

```diff
@@ -106,16 +112,44 @@ In a _pattern_, the resolved value of an _expression_ or _markup_ is used in its
 The form that resolved values take is implementation-dependent,
 and different implementations MAY choose to perform different levels of resolution.
 
-> For example, the resolved value of the _expression_ `{|0.40| :number style=percent}`
-> could be an object such as
+> While this specification does not require it,
+> a _resolved value_ could be implemented by requiring each function implementation to
+> return a value matching the following interface:
```

Comment:
I would make this more of a normal example. Perhaps:

```suggestion
> [!NOTE]
> This is an illustration of how an implementation might handle these requirements.
>
> An implementation might provide an interface similar to the following
> that each _function_ is required to return:
```

Reply:
I'm fine with rephrasing this, but two thoughts:
1. The "This is an illustration..." phrase seems a bit out of place here.
2. Saying that this is what a _resolved value_ could look like is kinda the key point here, and should not be lost.

Reply:
@aphillips I'm closing this thread as it's outdated, but please open a new one if you think that something here is still actionable.